### PR TITLE
Ui protractor

### PR DESCRIPTION
**Pull request by Varunc1**
Changed protractor test cases for login, partner, tenants and user mapping on dev environment

### CODE REVIEW

File: e2e/protractor.conf.js [hunk 1]
```diff
@@ -10,14 +7,14 @@ exports.config = {
...
+    './src/Tenants/Tenants.e2e-spec.ts',
+    './src/UserMapping/UserMapping.e2e-spec.ts'
   ],
   capabilities: {
-    'browserName': 'chrome'
+    'browserName': 'chrome',
   },
   directConnect: true,
-  baseUrl: 'https://presencesync.dev.niceincontact.com',
+  baseUrl: 'http://localhost:4200/login',
```
Comment (saiprasad1997) [inline e2e/protractor.conf.js]: don't push local changes

File: e2e/src/Partner/Partner.e2e-spec.ts [hunk 2]
```diff
@@ -44,21 +73,132 @@ xdescribe('Partner Test Cases', () => {
...
+    expect(partnerPage.enterEmail('somi.goyal@nice.com'));
+    expect(partnerPage.isValidateRegisterButtonPresent()).toBeTruthy('The Create and Activate button should be present');
+    expect(partnerPage.clickValidateRegisterButton());
+    expect(partnerPage.getModalError());
+    browser.wait(EC.browser.driver.sleep(1000));
+    expect(partnerPage.getModalClose());
+    testDone();
+  });
+  using(seleniumJsFlavours.PartnerCocaCola, function (data) {
+   /* it('should validate if Authentication URI is incorrect, display error message pop-up', function (testDone) {
```
Comment (saiprasad1997) [inline e2e/src/Partner/Partner.e2e-spec.ts]: remove commented code if not needed

File: e2e/src/Partner/Partner.po.ts [hunk 3]
```diff
@@ -24,6 +24,9 @@ export class PartnerPage {
     return this.getPartnerPage().isPresent();
   }
 
+  getPartnerTitle(){
```
Comment (saiprasad1997) [inline e2e/src/Partner/Partner.po.ts]: try to use id to find element wherever possible

File: e2e/src/Tenants/Tenants.e2e-spec.ts [hunk 4]
```diff
@@ -1,47 +1,71 @@
...
 import {Base} from '../base.po';
-
-xdescribe('BU Test Cases', () => {
+import {browser, protractor} from 'protractor';
+import {Login} from '../login/Login.po';
+let EC = protractor.ExpectedConditions;
+describe('BU Test Cases', () => {
   let buPage: BuPage;
-  const baseObj = new Base();
+  const LoginPage: Login = new Login();
```
Comment (saiprasad1997) [inline e2e/src/Tenants/Tenants.e2e-spec.ts]: variable name should start with small case

File: e2e/src/Partner/Partner.po.ts [hunk 5]
```diff
@@ -160,6 +163,26 @@ export class PartnerPage {
...
+    return element(by.css('#create-partner-cancel-button'));
+  }
+  isPartnerCancelButtonPresent(): promise.Promise<boolean> {
+    return this.getPartnerCancelButton().isPresent();
+  }
+  clickPartnerCancelButton() {
+    return this.getPartnerCancelButton().click();
+  }
+  getModalClose() {
+    return element(by.xpath('//button[contains(text(),\'Ok\')]')).click();
```
Comment (Susmith-roy-incontact) [inline e2e/src/Partner/Partner.po.ts]: use id here

File: e2e/src/Tenants/Tenants.e2e-spec.ts [hunk 6]
```diff
@@ -1,47 +1,71 @@
 import {BuPage} from './Tenants.po';
 import {Base} from '../base.po';
-
-xdescribe('BU Test Cases', () => {
+import {browser, protractor} from 'protractor';
+import {Login} from '../login/Login.po';
+let EC = protractor.ExpectedConditions;
```
Comment (Susmith-roy-incontact) [inline e2e/src/Tenants/Tenants.e2e-spec.ts]: convert to lowercase

File: e2e/protractor.conf.js (hunk 1 above)
Comment (Varunc1) [reply e2e/protractor.conf.js]: Done

File: e2e/src/Partner/Partner.e2e-spec.ts (hunk 2 above)
Comment (Varunc1) [reply e2e/src/Partner/Partner.e2e-spec.ts]: Test cases is valid it is commented because of bug that needs to be fixed by developers

**Pull request merged**